py-rse/bash-basics.Rmd

```diff
@@ -252,10 +262,10 @@ $ ls -F /Users
 Amira/  Jun/    Sami/
```

Comment:
Can we make these lowercase throughout the book (to match what's written in the narrative up above)? Not sure how other chapters list them

Reply:
good catch - fixed.